### PR TITLE
Integrations/add calculators

### PR DESCRIPTION
**Pull request by aapomm**
### Spec
This PR adds the CVSS Calculator to the Tools Manager and removes the `show` setting.

### Testing Steps

Provide steps to test functionality, described in detail for someone not familiar with this part of the application / code base

### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here. This could include
benchmarks, or other information.

Thanks for contributing to Dradis!

### Copyright assignment

Collaboration is difficult with commercial closed source but we want
to keep as much of the OSS ethos as possible available to users
who want to fix it themselves.

In order to unambiguously own and sell Dradis Framework commercial
products, we must have the copyright associated with the entire
codebase. Any code you create which is merged must be owned by us.
That's not us trying to be a jerks, that's just the way it works.

You can delete this section, but the following sentence needs to
remain in the PR's description:

> I assign all rights, including copyright, to any future Dradis
> work by myself to Security Roots.

### Check List

- [x] Added a CHANGELOG entry
- [ ] Added specs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+v4.13.0 (Month 2024)
+  - Add calculator to the Tools Manager
```
Comment (caitmich) [inline CHANGELOG.md:2]:
```suggestion
  - Add ability to enable/disable calculator in the Tools Manager
```

File: lib/dradis/plugins/calculators/cvss/engine.rb [hunk 2]
```diff
@@ -26,10 +22,17 @@ class Engine < ::Rails::Engine
     end
 
     initializer 'calculator_cvss.mount_engine' do
-      Rails.application.routes.append do
-        mount Dradis::Plugins::Calculators::CVSS::Engine => '/', as: :cvss_calculator
+      Rails.application.reloader.to_prepare do
+        if (ActiveRecord::Base.connection rescue false) && ::Configuration.table_exists?
```
Comment (etdsoft) [inline lib/dradis/plugins/calculators/cvss/engine.rb:26]: Why is this needed? I don't see it in other integrations?

Comment (aapomm) [reply lib/dradis/plugins/calculators/cvss/engine.rb:26]: See: https://github.com/dradis/dradis-html_export/blob/main/lib/dradis/plugins/html_export/engine.rb#L26. This is causing timing issues during OVA building as this addong is loaded when the main app is initialized.

Comment (aapomm) [reply lib/dradis/plugins/calculators/cvss/engine.rb:29]: Added comment for future reference ✅
